specification/devtestlabs/resource-manager/Microsoft.DevTestLab/stable/2018-09-15/DTL.json

```diff
@@ -2545,7 +2489,6 @@
             "name": "$expand",
             "in": "query",
             "description": "Specify the $expand query. Example: 'properties($select=status)'",
-            "required": false,
```

Comment:
Breaking changes need ARM review

Reply:
For query parameters, the default value for 'required' is false. So removing it does not change the behavior.

Reply:
Yeah, I don't see how this would be breaking?